pyaedt/hfss3dlayout.py

```diff
@@ -363,6 +363,24 @@ def create_wave_port_from_two_conductors(self, primivitivenames=[""], edgenumber
         else:
             return False
 
+    @pyaedt_function_handler()
+    def dissolve_component(self, component_name):
+        """Dissolve a component and remove it from 3D Layout.
+
+        Parameters
+        ----------
+        component_name : str
+            Name of the components.
```

Comment:
Both the method description and the parameter name imply a singular component. I think the parameter description should also use the singular form.